modules/plugin/imagepyramid/src/main/java/org/geotools/gce/imagepyramid/ImagePyramidFormat.java

```diff
@@ -206,20 +206,15 @@ public boolean accepts(Object source, Hints hints) {
             }
```

Comment:
Shouldn't this one advertise support for the overview policy now? Generally speaking, I'd change the code so that it advertises support for all the mosaic params (calling onto the mosaic format to grab them first)